Automattic/wp-calypso
# Signup: Update themes in signup to replace seven with new ones

This PR updates seven themes in sign up with new ones, and also updates three of those themes in the "You might also like..." section that appears at the bottom of the theme sheets.

**Steps to test locally:**

1. Switch to this PR.
2. The easiest way to test the new themes in signup is to look at a couple specific flows:
* Go to http://calypso.local:3000/start/portfolio - refresh the page to make sure the following themes are there:
	* Publication (instead of Boardwalk)
	* Canard (instead of Rowling)
	* Illustratr (instead of Baskerville 2)
	* Gazette (instead of Cubic)
* Go to http://calypso.localhost:3000/start/personal/themes - refresh to make sure you've seen the following:
	* Libre 2 (instead of Hemingway Rewritten)
	* Libretto (instead of Cols)
	* Intergalactic 2 (instead of Bushwick)
3. To test the recommended themes, you will need to visit multiple theme sheets (eg. http://calypso.localhost:3000/theme/radcliffe-2) and scroll to the bottom to view the screenshots under 'You might also like'. Look for:
	* Libretto (instead of Cols)
	* Integalactic 2 (instead of Bushwick)
	* Libre 2 (instead of Hemingway Rewritten)

Also testing can be done here: https://calypso.live/?branch=themes-in-signup-general

Thanks in advance!
